aemy-prompts.md

```diff
@@ -251,15 +253,6 @@ Import content
 start importing content for the site
 ```
 
-#### With Options
```

Comment:
Are we discouraging this approach?

Reply:
Yes - we do not want options specified in the prompt. Either use the defaults or add an `aemy.json`.